etc/hooks/pre-commit.sh

```diff
@@ -5,7 +5,8 @@ FILES=$(git diff --cached --name-only --diff-filter=ACMR | sed 's| |\\ |g')
 [ -z "$FILES" ] && exit 0
 
 # Prettify all selected files
-echo "$FILES" | xargs ./pants fmt
+# (--owners-not-found-behavior=ignore will skip changed files that are not Pants-managed)
+echo "$FILES" | xargs ./pants --owners-not-found-behavior=ignore fmt
```

Comment:
I think all this could be simplified by having Pants decide what has changed: https://www.pantsbuild.org/docs/reference-changed. No `xargs`, no ignoring of files.